shell/platform/embedder/embedder.h

```diff
@@ -572,6 +572,8 @@ typedef struct {
   size_t left;
   /// Vertical physical location of the top of the window on the screen.
   size_t top;
+  /// Bottom inset of window.
+  double physical_view_inset_bottom;
```

Comment:
I suggest adding all 4 inset parameters (top, left, bottom, right) while you're at it.

Reply:
Done.